PipelinesTasks/PRMetrics/buildAndReleaseTask/src/metrics/codeMetrics.ts

```diff
@@ -0,0 +1,277 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+import { FixedLengthArray } from '../utilities/fixedLengthArray'
+import { ICodeFileMetric } from './iCodeFileMetric'
+import { singleton } from 'tsyringe'
+import * as taskLib from 'azure-pipelines-task-lib/task'
+import CodeMetricsData from './codeMetricsData'
+import GitInvoker from '../git/gitInvoker'
+import Inputs from './inputs'
+import TaskLibWrapper from '../wrappers/taskLibWrapper'
+
+/**
+ * A class for computing metrics for software code in pull requests.
+ */
+@singleton()
+export default class CodeMetrics {
+  private _gitInvoker: GitInvoker
+  private _inputs: Inputs
+  private _taskLibWrapper: TaskLibWrapper;
```

Comment:
I see here you're adding semi-colon, and elsewhere you're not. Might be best to keep consistent across repo?

Reply:
Good catch. The ESlint standard style (which we use here) says not to use semicolons unless truly required. And they're not required here, so they definitely should go...